temporalcli/commands.taskqueue.go

```diff
@@ -11,31 +11,237 @@ import (
 	enumspb "go.temporal.io/api/enums/v1"
 	"go.temporal.io/api/taskqueue/v1"
 	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
 	"go.temporal.io/server/common/tqid"
 )
 
+const unversioned = "UNVERSIONED"
```

Comment:
Where is this constant used?

Reply:
The unversioned queue is represented with an empty string, this is just a constant to define a printable name. I'll make it more specific...